### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by mgwalker**
Refreshed some documentation

### This pull request changes...
- mainly the README
- just some code formatting changes in some of the other files

### This pull request is ready to merge when...
- [ ] This code has been reviewed by someone other than the original author

### This feature is done when...
- [ ] Product has approved the experience

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,62 +3,164 @@
 [![Known Vulnerabilities](https://snyk.io/test/github/18f/cms-hitech-apd/badge.svg?targetFile=web%2Fpackage.json)](https://snyk.io/test/github/18f/cms-hitech-apd?targetFile=web%2Fpackage.json)
 [![Known Vulnerabilities](https://snyk.io/test/github/18f/cms-hitech-apd/badge.svg?targetFile=api%2Fpackage.json)](https://snyk.io/test/github/18f/cms-hitech-apd?targetFile=api%2Fpackage.json)
 
-# CMS HITECH APD app
+# CMS eAPD app
 
 This project aims to create a user-friendly, modern product to allow states to
-submit their Medicaid HITECH APD information to CMS easily and help CMS analysts
-review requests more efficiently.
+submit their Medicaid APD information to CMS easily and help CMS analysts
```
Comment (jeromeleecms) [inline README.md]: This project aims to create a user-friendly, modern product to streamline the creation, submission, review and approval of Medicaid HITECH APDs and their associated contract documents.

What do you think about this?

**Comment by jeromeleecms** [comment]
Overall, changes look good. Do you want me to try and run through the updated manual steps before I approve this PR? I got stuck last time I tried this with the old documentation (but may be due to other internal restrictions).

**Comment by mgwalker** [comment]
That would be very helpful, if you've got time!

**Comment by jeromeleecms** [comment]
@mgwalker Does any of the feedback from @hbillings on her installation experience affect this doc? I can always test the installation instructions after the PR goes in.

**Comment by mgwalker** [comment]
Heather's feedback was a lot of the reason for updating the doc.  Hopefully I captured most of the problems she was having (because there was missing info 😳) here.

**Pull request merged**